values.yaml

```
@@ -6,7 +6,7 @@ image:
   # built from the most recent commit
   #
   # tag: latest
-  tag: "v4.2.10"
+  tag: "v4.2.11"
```

suggestion: I just noticed that this can be changed to `null`, as all template files are setup to fallback to `.Chart.AppVersion` if `.Values.image.tag` is not set.

```suggestion
  tag: null
```

Then future version bumps only need to modify `Chart.yaml` :)